internal/cmd/local/local/install.go

```diff
@@ -648,29 +648,44 @@ func (c *Command) handleChart(
 		}
 	}
 
-	pterm.Info.Println(fmt.Sprintf(
-		"Starting Helm Chart installation of '%s' (version: %s)",
-		req.chartName, helmChart.Metadata.Version,
-	))
-	c.spinner.UpdateText(fmt.Sprintf(
-		"Installing '%s' (version: %s) Helm Chart (this may take several minutes)",
-		req.chartName, helmChart.Metadata.Version,
-	))
-	helmRelease, err := c.helm.InstallOrUpgradeChart(ctx, &helmclient.ChartSpec{
-		ReleaseName:     req.chartRelease,
-		ChartName:       req.chartLoc,
-		CreateNamespace: true,
-		Namespace:       req.namespace,
-		Wait:            true,
-		Timeout:         60 * time.Minute,
-		ValuesYaml:      req.valuesYAML,
-		Version:         req.chartVersion,
-	},
-		&helmclient.GenericHelmOptions{},
-	)
-	if err != nil {
-		pterm.Error.Printfln("Failed to install %s Helm Chart", req.chartName)
-		return fmt.Errorf("unable to install helm: %w", err)
+	var helmRelease *release.Release
+
+	// it's possible that an existing helm installation is stuck in a non-final state
+	// which this code will detect, attempt to clean up, and try again up to three times
+	for attemptCount := 0; attemptCount < 3; attemptCount++ {
+		pterm.Info.Println(fmt.Sprintf(
+			"Starting Helm Chart installation of '%s' (version: %s)",
+			req.chartName, helmChart.Metadata.Version,
+		))
+		c.spinner.UpdateText(fmt.Sprintf(
+			"Installing '%s' (version: %s) Helm Chart (this may take several minutes)",
+			req.chartName, helmChart.Metadata.Version,
+		))
+
+		helmRelease, err = c.helm.InstallOrUpgradeChart(ctx, &helmclient.ChartSpec{
+			ReleaseName:     req.chartRelease,
+			ChartName:       req.chartLoc,
+			CreateNamespace: true,
+			Namespace:       req.namespace,
+			Wait:            true,
+			Timeout:         60 * time.Minute,
+			ValuesYaml:      req.valuesYAML,
+			Version:         req.chartVersion,
+		},
+			&helmclient.GenericHelmOptions{},
+		)
+
+		if err != nil {
+			if strings.Contains(err.Error(), "another operation (install/upgrade/rollback) is in progress") {
```

Comment:
Don't you need to check attempt count somewhere? I think you could check it here

Reply:
I don't need to check it here, but I need to check if it was successful outside of this loop.

Reply:
@abuchanan-airbyte I added a check after the for-loop to ensure that `helmRelease` was properly set.  I don't care what the `attemptCount` value is, I only care that we attempted more than once (after removing the helm-secrets on the first failed attempt) and less an an infinite amount.  I picked three attempts as it was one more than the number of attempts it should realistically take (i.e. two).

Additionally I added a new `localerr.ErrHelmStuck` error that will now display information to the end-user on how an to _fix_ this problem if the newly added (in this PR) functionality to _fix_ this problem for them fails.